src/server/scripts/Kalimdor/HallsOfOrigination/boss_setesh.cpp

```diff
@@ -0,0 +1,531 @@
+
+                // Walk towards chaos portal.
+                npcChaosPortal->CastSpell((Unit*)nullptr, SPELL_DUMMY_AURA, true);
+                me->GetMotionMaster()->MoveCloserAndStop(POINT_CHANNEL_CHAOS_PORTAL, npcChaosPortal, 18.0f);
+            }
+
+            void CleanUp()
+            {
+                std::list<Creature*> units;
+                GetCreatureListWithEntryInGrid(units, me, NPC_SETESH_CHAOS_BLAST, 200.0f);
+                for (std::list<Creature*>::iterator itr = units.begin(); itr != units.end(); ++itr)
+                    (*itr)->DespawnOrUnsummon();
+
+                units.clear();
+                GetCreatureListWithEntryInGrid(units, me, NPC_SETESH_CHAOS_SEED, 200.0f);
+                for (std::list<Creature*>::iterator itr = units.begin(); itr != units.end(); ++itr)
+                    (*itr)->DespawnOrUnsummon();
+            }
+        };
+
+        CreatureAI* GetAI(Creature* creature) const
+        {
+            return new boss_seteshAI(creature);
+        }
+};
+
+// 41041 Chaos Blast
+class npc_setesh_chaos_blast : public CreatureScript
+{
+public:
+    npc_setesh_chaos_blast() : CreatureScript("npc_setesh_chaos_blast") { }
+
+    struct npc_setesh_chaos_blastAI : public ScriptedAI
+    {
+        npc_setesh_chaos_blastAI(Creature* creature) : ScriptedAI(creature) { }
+
+        void Reset() override
+        {
+            me->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NOT_SELECTABLE);
+            if (InstanceScript* instance = me->GetInstanceScript())
+                if (Creature* setesh = instance->GetCreature(DATA_SETESH))
+                    setesh->CastSpell(me, SPELL_CHAOS_BLAST_MISSLE, true);
+        }
+
+        void SpellHit(Unit* /*caster*/, const SpellInfo* spellInfo) override
+        {
+            if (spellInfo->Id != SPELL_CHAOS_BLAST_MISSLE)
+                return;
+
+            DoCastSelf(SPELL_CHAOS_BLAST_AURA);
+        }
+    };
+
+    CreatureAI* GetAI(Creature* creature) const override
+    {
+        return GetHallsOfOriginationAI<npc_setesh_chaos_blastAI>(creature);
+    }
+};
+
+// 41126 Chaos Seed
+class npc_setesh_chaos_seed : public CreatureScript
+{
+public:
+    npc_setesh_chaos_seed() : CreatureScript("npc_setesh_chaos_seed") { }
+
+    struct npc_setesh_chaos_seedAI : public ScriptedAI
+    {
+        npc_setesh_chaos_seedAI(Creature* creature) : ScriptedAI(creature)
+        {
+            me->SetDisableGravity(true);
+            me->SetFloatValue(UNIT_FIELD_BOUNDINGRADIUS, 7.0f);
+            me->SetFloatValue(UNIT_FIELD_COMBATREACH, 7.0f);
+            me->setFaction(FACTION_FRIENDLY);
+
+            DoCastSelf(SPELL_SEED_OF_CHAOS_DUMMY);
+        }
+
+        void Reset() override
+        {
+            // Simulate slow fall
+            float x = me->GetPositionX();
+            float y = me->GetPositionY();
+            float z = me->GetPositionZ();
+            Position dest = { x, y, me->GetMap()->GetWaterOrGroundLevel(me->GetPhases(), x, y, z) };
```

Comment:
```cpp
src/server/scripts/Kalimdor/HallsOfOrigination/boss_setesh.cpp:343:77:
 fatal error: no member named 'GetPhases' in 'Creature'
            Position dest = { x, y, me->GetMap()->GetWaterOrGroundLevel(me->GetPhases(), x, y, z) };
                                                                        ~~  ^
1 error generated.
```

Reply:
GetPhaseShift() since the phasing rewrite